src/app/modules/form/admin-form/admin-form.utils.ts

```diff
@@ -62,6 +63,11 @@ export const mapRouteError = (
         statusCode: StatusCodes.UNPROCESSABLE_ENTITY,
         errorMessage: error.message,
       }
+    case TransferOwnershipError:
+      return {
+        statusCode: StatusCodes.CONFLICT,
```

Comment:
shouldn't this be 400? transferownershiperror means they tried to transfer to the existing owner or a nonexistent user right?